src/comm/tests/mpsc.rs

```diff
@@ -80,7 +80,7 @@ fn test_mpsc_limited_close() -> Result<(), Box<dyn Error>> {
 /// AddrNotAvailable error because the OS will run out of outgoing TCP ports.
 #[test]
 fn test_mpsc_connection_reuse() -> Result<(), Box<dyn Error>> {
-    ore::log::init();
+    ore::test::init_logging();
```

Comment:
test?

Reply:
aka I'd be tempted to move this up to `ore::log` instead of moving `ore::log` into `ore::test` but 🤷🏻‍♂️

Reply:
Yeah I wanted to highlight that it's only meant to be called in tests, since it doesn't do the right thing re: choosing between a file and stderr for actual binaries. Can definitely revisit if this doesn't make sense down the line.